### PR TITLE
Feature/jobstreet footer localisation

### PR DESCRIPTION
**Pull request by mengtzu**
Opening the PR partially as a discussion piece - while we may be OK to merge as is, this is more a proposal for handing i8n within complex compound components, before we try applying it to the headers.

### CODE REVIEW

**Comment by mengtzu** [comment]
Note I chose this approach to match CPP, and with the assumption that the formatting features of `react-intl` are useful.

If we don't value either of those things, we could very easily remove `react-intl` and just use the message objects directly.  I think @nicolassenechal supported this idea?

**Comment by mengtzu** [comment]
OK yeah, I hated `react-intl` being in here, the footer is just too simple to benefit.  Alternate implementation pushed.

File: docs/src/components/Playground/Playground.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ export default class Playground extends Component {
     return (
       <div>
         <JobStreetHeader user={{}} />
-        <JobStreetFooter />
+        <JobStreetFooter language="vi" country="vn" />
```
Comment (nicolassenechal) [inline docs/src/components/Playground/Playground.js]: Any reason why we would hard-code here?

Comment (mengtzu) [reply docs/src/components/Playground/Playground.js]: As discussed, we don't care what happens in /playground, it's a dev convenience.

This does highlight the need for a demo page, I'll add one

**Pull request merged**